deeplake/enterprise/convert_to_libdeeplake.py

```diff
@@ -145,6 +145,11 @@ def dataset_to_libdeeplake(hub2_dataset):
             raise ValueError("In memory datasets are not supported for libdeeplake")
         elif path.startswith("hub://"):
             token = hub2_dataset._token
+            if token is None or token == "" and not hub2_dataset.public:
+                token = hub2_dataset.client.get_token()
+                if token is None or token == "":
+                    raise EmptyTokenException
+
```

Comment:
Do we need same validations of empty token in s3 and in the else portion of the code i.e. wherever token is used. 


`            token = hub2_dataset._token
            org_id = hub2_dataset.org_id
            if token is None:
                libdeeplake_dataset = api.dataset(path)
            elif token == "":
                raise EmptyTokenException
            else:
                org_id = (
                    org_id
                    or jwt.decode(token, options={"verify_signature": False})["id"]
                )
                libdeeplake_dataset = api.dataset(path, token=token, org_id=org_id)

        hub2_dataset.libdeeplake_dataset = libdeeplake_dataset`